target/linux/realtek/files-5.10/arch/mips/rtl838x/prom.c

```diff
@@ -35,11 +35,16 @@ static struct plat_smp_ops rtl_smp_ops;
 
 static void rtl_init_secondary(void)
 {
+#ifndef CONFIG_CEVT_R4K
 /*
- * MIPS timer interrupt might fire like crazy if not used or initialized
- * properly. Silence it by setting the maximum possible interval.
+ * These devices are low on resources so it is helpful to disable all unneeded
+ * features. If CEVT_R4K is not enabled we have no registered handler for the
+ * internal timer interrupt 7 that may be active by default. In this case
+ * disable counting in the core and confirm a pending interrupt.
  */
+	write_c0_cause(read_c0_cause() | BIT(27));
```

Comment:
The described behaviour matches the definition of `BIT(27)` or `CAUSEF_DC` in the kernel's mipsregs.h. Please use the latter instead, since that's more descriptive. Not that I could find that much information, since it appears to be reserved on R4K.

Reply:
See https://s3-eu-west-1.amazonaws.com/downloads-mips/documents/MD00534-2B-34K-SUM-01.13.pdf page 203

Reply:
done